src/firebase/fixUserData.js

```diff
@@ -0,0 +1,128 @@
+// Script to fix data corrupted by Issue #148 on GitHub
+// NOTE: This is only able to work as the Fall 2022 and Ongoing semesters as of making this has no overlapping programs
+// Fall 2022 and Ongoing are the only semesters with displayed applications
+// Employment Opporutnities (the only program of Ongoing sem) data is completely lost and cannot be restored
+const firebase = require("firebase/app");
```

Comment:
This script runs without any credentials. It will fail if we enforce the security rules on firestore. I would suggest using `firebase-admin` instead, with credentials of a service account. See [https://firebase.google.com/docs/admin/setup](https://firebase.google.com/docs/admin/setup).
Note the `apiKey` in the `config` is for public access... You still need to log in so as to make changes on db, just as a user logging in the Applicant Portal, which requires OAuth.
Also it would be better to move the script outside `src` directory... It's not a part of the frontend.

Reply:
I have moved the file into the scripts folder. Also, I don't think we will have to run this script again, do you think I should remove the file from the repository?

Reply:
Okay I guess that would be fine.